tests/Functional/Language/ParserTest.php

```diff
@@ -568,4 +561,25 @@ public function testParsesNestedTypes()
             ],
         ]);
     }
+
+    /**
+     * The purpose of this test case is that it should *not* crash with "Syntax Error: Cannot contain the invalid character <EOF>"
+     * @throws SyntaxErrorException
+     * @throws \Digia\GraphQL\Error\InvariantException
+     */
+    public function testParsesGitHubIssue253(): void
+    {
+        $document = parse(dedent('{
+  songs(first: 5, search: {name: "Vårt land"}) {
+    edges {
+      node {
+        name
+      }
+    }
+  }
+}
+        '));
+
+        $this->addToAssertionCount(1);
```

Comment:
This is a bit lame but it's the only way to test for "does not throw an exception" AFAIK